### DISCUSSION TITLE
Is it safe to copy and paste old config file(config\tweakeroo.json) to newer version?

### DISCUSSION THREAD

**Discussion by ShinhuPark** [maruohon/tweakeroo#304] [Q&A] [answered]
Since Tweakeroo specially have much more hotkey binds then other malilib mods, I didn't wanna get bothered setting keybinds from square one, so everytime Tweakeroo updates along with newer minecraft version, I copied and pasted config file(config\tweakeroo.json) from older Tweakeroo version into newer Tweakeroo version.

This haven't caused any issue, until now the 1.18 update.

When I copied config files from 1.17 Tweakeroo to 1.18 Tweakeroo, everything works fine except the "tweakFakeSneakPlacement". 

The issue was that even when FakeSneakPlacement is off, I couldn't strip logs with right clicking with axe. I could still right click on chest and open the GUI, every right click functions work except stripping logs.

Was what I have been doing so far(copy and pasting the old config file) wrong?

Here's the config file that caused the issue described above(I changed the extension from json to txt)
[tweakeroo.txt](https://github.com/maruohon/tweakeroo/files/7780059/tweakeroo.txt)

**Comment by maruohon** [comment, marked as answer]
Copying the config is fine. Sometimes a few config options might get added or changed, but that doesn't affect the other configs.

As to your issue, you seem to have the `disableAxeStripping` feature enabled.
![image](https://user-images.githubusercontent.com/2331050/147476962-bfb3fb67-232b-4576-a332-7e9cb592ac21.png)

**Reply by ShinhuPark** [reply]
Oh my gosh.. In  the past I must have tweaked this setting but didn't remember..
Thank you so much for your help. This issue was all because of me 😓
